public/js/signin.js

```diff
@@ -15,7 +15,7 @@ const signinFormHandler = async (event) => {
         });
 
         if (response.ok) {
```

Comment:
This redirect will happen on successful login from the user routes: 
![image](https://user-images.githubusercontent.com/9697002/204113378-93476a88-9550-4d2f-8804-a20d852a5095.png)

Reply:
This is incorrect, please disregard